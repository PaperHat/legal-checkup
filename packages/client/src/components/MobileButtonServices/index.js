import React from 'react'
import StyledMobileButton from '../MobileButton'

export default function MobileButtonServices () {
  return (
    <StyledMobileButton
      onClick={() => {
        window.open('http://www.wclc.org.nz/our-services/', '_blank')
      }}
    >
      Community Law Services
    </StyledMobileButton>
  )
}
