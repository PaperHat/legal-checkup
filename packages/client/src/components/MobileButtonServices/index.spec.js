import React from 'react'
import { shallow } from 'enzyme'
import MobileButtonServices from '.'

describe('components:MobileButtonServices', () => {
  it('renders the MobileButtonServices and CSS properties properly', () => {
    expect(toJson(shallow(<MobileButtonServices />))).toMatchSnapshot()
  })
})
