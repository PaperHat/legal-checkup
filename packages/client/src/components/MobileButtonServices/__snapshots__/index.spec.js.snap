// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:MobileButtonServices renders the MobileButtonServices and CSS properties properly 1`] = `
<StyledMobileButton
  onClick={[Function]}
>
  Community Law Services
</StyledMobileButton>
`;
